src/index.css

```diff
@@ -9,14 +9,77 @@ body {
 }
 
 ul {
+  list-style: none;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+#menu {
   display: flex;
   gap: 20px;
-  list-style: none;
+  text-transform: capitalize;
+}
+
+#list-block {
+  display: grid;
+  gap: 25px;
+  grid-template-columns: 1fr;
+}
+
+#list-block img {
+  width: 100%;
+}
+
+#popup {
+  display: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  inset-inline: 0;
+  padding: 10px;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+#popup-block {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  background-color: #fff;
+  margin: auto;
+}
+
+#popup-close {
+  align-self: flex-end;
+}
+
+#popup-block form {
```

Comment:
what about using `classes` instead of ids while trying to apply `css` rules to you `html` ?

It's one among CSS best practices recommendation

Reply:
Nice suggestion 👌🏻